src/app/editors/contentlist/contentlist.component.ts

```diff
@@ -49,6 +44,9 @@ export class ContentlistComponent implements OnInit {
         }
       }
     };
+    if (_.get(this.userData, 'role') === 'creator') {
```

Comment:
@santoshilimi  You have passsed it for creator but what about reviewer workflow ?
Please pass it below field also when user selected  role as reviewer:
```
createdBy: {!=: "<USER_ID>"}

```

Reply:
Done